test/e2e/installplan_e2e_test.go

```diff
@@ -107,26 +102,21 @@ var _ = Describe("Install Plan", func() {
 					counter = metric.Value
 				}
 			}
+			deprecatedCRD, err := util.DecodeFile("./testdata/deprecated-crd/deprecated.crd.yaml", &apiextensionsv1.CustomResourceDefinition{})
```

Comment:
There's a CLI flag for determining the testdata base directory: `testdataPath`.

Reply:
Thanks!